Commands/SynchronizeUsers.php

```diff
@@ -52,6 +52,8 @@ protected function configure()
         $this->addNoValueOption('skip-existing', null,
             "Skip users that have been synchronized at least once. Using this option will be much faster, but will not "
             . "update user info if it has changed in LDAP.");
+        $this->addNoValueOption('purge-non-existent-users', null,
+            "Purge old users from Matomo, which are removed in LDAP.");
```

Comment:
Maybe the message could be clearer:

1. Purge users from Matomo that no longer exist in LDAP.
2. Identify users that have been removed from LDAP and purge them from Matomo.